include/deal.II/dofs/dof_accessor.h

```diff
@@ -2019,6 +2019,19 @@ class DoFCellAccessor : public DoFAccessor<DoFHandlerType::dimension,
   void
   clear_p_coarsen_flag() const;
 
+  /**
+   * Returns the active_fe_index that this cell would have if refinement would
+   * have happened.
+   *
+   * Prediction is based on the current p-refinement flags set on this cell,
+   * returning FECollection::next_in_hierarchy() for p-refinement,
+   * FECollection::previous_in_hierarchy() for p-coarsening,
+   * or the current active_fe_index otherwise.
```

Comment:
Can you say what happens if a cell is also marked for h-refinement or coarsening?

Reply:
I updated the comment. This function will be obsolete in a future PR.